docs/docusaurus/docs/conceptual_guides/metricproviders.md

```diff
@@ -0,0 +1,105 @@
+---
+sidebar_label: 'MetricProviders'
+title: 'MetricProviders'
+id: metricproviders
+description: Learn how MetricProviders are an integral component of the Expectation software development kit (SDK).
+---
+
+MetricProviders generate and register Metrics to support Expectations, and they are an important part of the Expectation software development kit (SDK). Typically, MetricProviders are used to resolve complex requirements and are not implemented widely. If you're considering using MetricProviders, you're encouraged to join the [Great Expectations Slack community](https://greatexpectations.io/slack) and discuss your requirements in the [#gx-community-support](https://greatexpectationstalk.slack.com/archives/CUTCNHN82) channel.
+
+You'll encounter references to MetricProviders in many of the topics in the Great Expectations (GX) documentation. An in-depth knowledge of MetricProvider functionality is helpful, but not required to successfully complete the tasks documented in these topics. However, some knowledge of MetricProvider functionality is useful if you're developing Custom Expectations, or you're maintaining or extending MetricProvider classes.
+
+MetricProviders let you declare all the Metrics that are needed to support an Expectation in concise, Don't Repeat Yourself (DRY) syntax. MetricProviders support the following use cases:
+
+- Conceptually grouping code for Metrics calculation across multiple backends.
+- Incremental development. You can develop Metrics for each Execution Engine, one at a time.
+- Sharing Metrics across Expectations.
+- Generating Metrics that can be inferred from specific types of Expectation automatically.
+
+## Assumed knowledge
+
+To get the most out of the information provided here, you should have an understanding of:
+
+- Expectations
+- Metrics
+- The Metric registry
+- ExecutionEngines
+
+## When MetricProviders are required
+
+A minimum of one supporting Metric is required by every Expectation. For example, `expect_column_mean_to_be_between` relies on a Metric that calculates the mean of a column. Often, an Expectation requires multiple Metrics. For example, the following Metrics are required in the `expect_column_values_to_be_in_set` Expectation:
+
+- `column_values.in_set.unexpected_count`
+- `column_values.in_set.unexpected_rows`
+- `column_values.in_set.unexpected_values`
+- `column_values.in_set.unexpected_value_counts`
+
+To allow Expectations to work with multiple backends, methods for calculating Metrics need to be implemented for each ExecutionEngine. For example the `aggregate_fn` utility that is used with SQL and Spark requires the `column_values.in_set.unexpected_count.aggregate_fn` Metric. 
+
+Metrics can help you incorporate conditional statements in Expectations that support conditional evaluations. For example, `column_values.in_set.condition`.
```

Comment:
We have the name of a metric here, but how is it used to incorporate conditional statements in Expectations?  Maybe expand this to be something like: "For example, `column_values.in_set.condition` can be set when metrics are calculated and then used as a flag in your Custom Expectation to determine branching behavior."  (Assuming that's an accurate explanation of the example.  I'd double check with someone more familiar with Custom Expectations.)